index.html

```diff
@@ -10711,7 +10711,47 @@ <h2>Definitions of States and Properties (all aria-* attributes)</h2>
 				</tbody>
 			</table>
 		</div>
-                <div class="property" id="aria-details">
+		<div class="property" id="aria-description">
+			<pdef>aria-description</pdef>
+			<div class="property-description">
+				<p>Defines a string value that describes or annotates the current element. See related <pref>aria-describedby</pref>.</p>
+				<p>The <pref>aria-description</pref> attribute is similar to the <pref>aria-label</pref> in that both provide a flat string to associate with the element, but a label should be concise, where a description is intended to provide more verbose information.</p>
```

Comment:
```suggestion
				<p>The <pref>aria-description</pref> attribute is similar to the <pref>aria-label</pref> in that both provide a flat string to associate with the element, but a label should be concise, whereas a description is intended to provide more verbose information.</p>
```

Consider using whereas instead of where.